modules/ossm-rn-new-features.adoc

```diff
@@ -381,6 +428,28 @@ Kiali 1.36 includes the following features and enhancements:
 * Namespace and cluster boxing to support federated service mesh views
 * New validations, wizards, and distributed tracing enhancements
 
+== New features {SMProductName} 2.0.12
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), and is supported on OpenShift Container Platform 4.9 or later.
+
+=== Component versions included in {SMProductName} version 2.0.12
```

Comment:
Verify version, Is it 2.0.12 or 2.0.11.1?

Reply:
@jshaughn the PR has been updated based on the information discussed in the program call. The revised preview build is also available. Thanks for bringing this up so that it could be clarified.